<!-- -->
<launch>
	<!--can分析仪驱动-->
	<node name="usbcan_driver_node" pkg="usbcan_driver" type="usbcan_driver_node" output="screen">
		<rosparam param="baudrate" > [500,500] </rosparam>
		<rosparam param="frame_id" > ["channel1","channel2"] </rosparam> <!--通道1读取和发送，通道2不连接-->
		<!--配置滤波器 通道1全部接收; 通道2全不接收-->
		<rosparam param="acc_code" >  [0xFFFFFFF,0x0] </rosparam>
		<rosparam param="mask_code" > [0xFFFFFFF,0x0] </rosparam>
		<param name ="from_can_topic"  type ="string"  value="/ars_radar/from_can_topic"/>
		<param name ="to_can_topic"    type ="string"  value="/ars_radar/to_can_topic"/>
	</node>
	
	<!--毫米波雷达驱动-->
	<node name="ars_radar_node" pkg="ars_radar" type ="ars_radar_node" output="screen">
		<param name ="is_sendMsgToArs" type ="bool"  value="True"/>
		<param name ="from_can_topic"  type ="string"  value="/ars_radar/from_can_topic"/>
		<param name ="to_can_topic"    type ="string"  value="/ars_radar/to_can_topic"/>
		<param name="gpsUtm_topic"    type ="string"  value="/gpsUtm"/>
	</node>
	
	<!-- 运行rviz可视化界面 -->
	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find ars_radar)/config/view_in_rviz.rviz" required="true" />
	
<!--	<node name="record_ars_node" pkg="ars_radar" type="record_ars_node" output= "screen">-->
<!--		<param name="file_path" value="$(find ars_radar)/data/"/>-->
<!--		<param name="file_name" value="ars_radar"/>-->
<!--	</node>-->

</launch>
